### ISSUE TITLE
Allow items to have multiple files uploaded.

### ISSUE THREAD

**Issue by kgf** [ReadyResponder/ReadyResponder#507]
This may need to be split into two PRs. We need to be able to upload multiple photos for an item. We also want to be able to upload documentation, such as owner's manuals.

**Comment by kgf** [comment]
To clarify the statement above, I would like to know which files uploaded are photos, and which are documentation.

**Comment by radvc** [comment]
@kgf so we need to have another table for this one?

**Comment by kgf** [comment]
@radvc Let's start with allowing multiple photos, rather than just the one we have now. We can look at the documentation uploads later. That should perhaps be another issue. 

 If adding multiple photos requires a different approach, please leave the existing photos in addition to the ones you'll add. I'm thinking there would be one photo that is the thumbnail, and then others showing other details.

**Comment by radvc** [comment]
is there a maximum number of photos?

**Comment by kgf** [comment]
Not that immediately comes to mind. I would not expect to have more than 20...

**Comment by radvc** [comment]
okay so we need to have new table for this one, item has_many item_images?

**Comment by kgf** [comment]
It seems likely, although I don't actually know. If that's the case, maybe we can leave the existing image as is, and that will be the gallery image.
I'm considering the original request too. Is having two file upload mechanisms, one for images and one for documentation, a lot of effort for not enough reward?

**Comment by radvc** [comment]
I can do that, are we going to separate the file input tag for image and documentation?

**Comment by kgf** [comment]
I was thinking we would originally, but it may not be worth the effort.. Leave the existing photo upload, which will be used as a thumbnail for the item if needed.

**Comment by kgf** [comment]
@radvc Did you have any additional questions ? Are you having any success ?

**Comment by kgf** [comment]
@radvc Did you have any success with this ? Does it require another table ?

**Comment by kgf** [comment]
@radvc, let me know if you're still interested in this, I'll add the Claimed label back on...

**Comment by vsrnth** [comment]
![image](https://user-images.githubusercontent.com/6362618/46524711-5373ed00-c8a7-11e8-8dd3-94d791a42175.png)

http://localhost:3000/item_types/new
The upload button at this page needs to support multiple image upload?

**Comment by kgf** [comment]
That is correct. It will probably need to be another class that belongs_to item.

**Comment by kgf** [comment]
@vsrnth I have changed the label to claimed, based on your questions. I am hoping this is correct !
Thanks for your help.
Kevin

**Comment by vsrnth** [comment]
Yup I'm taking this on.
